cmd/cli/job/describe.go

```diff
@@ -156,6 +188,33 @@ func (o *DescribeOptions) printExecutions(cmd *cobra.Command, executions []*mode
 	return output.Output(cmd, executionCols, tableOptions, executions)
 }
 
+func (o *DescribeOptions) printHistory(cmd *cobra.Command, label string, history []*models.JobHistory) error {
+	if len(history) < 1 {
+		return nil
+	}
+
+	firstTime := history[0].Occurred()
+	timeSinceCol := output.TableColumn[*models.JobHistory]{
+		ColumnConfig: table.ColumnConfig{Name: historyTimeCol.ColumnConfig.Name, WidthMax: 10, WidthMaxEnforcer: text.WrapText},
+		Value:        func(h *models.JobHistory) string { return h.Occurred().Sub(firstTime).String() },
+	}
+
+	tableOptions := output.OutputOptions{
+		Format:  output.TableFormat,
+		NoStyle: true,
+	}
+	jobHistoryCols := []output.TableColumn[*models.JobHistory]{
+		timeSinceCol,
+		historyRevisionCol,
+		historyStateCol,
+		historyTopicCol,
+		historyEventCol,
+		historyDetailsCol,
+	}
```

Comment:
to avoid having a sparse column, it might make sense to show the details under the the even message